### DISCUSSION TITLE
-bash: syntax error near unexpected token `newline'

### DISCUSSION THREAD

**Discussion by corystardust** [iraf-community/iraf#144] [Q&A] [answered]
Hello
I installed iraf in my macOs 10.15.5, the process went well with no errors, but when I try to configure, the output is: "-bash: syntax error near unexpected token `newline'", this is what I'm typing:

```
===================================================================
====================  Installation Completed With No Errors  ==================
=====================================================================
 $ export PATH=/path/to/iraf/bin/:$PATH
 $make <x86_64>   %the operating system is 64 bits
-bash: syntax error near unexpected token `newline'
```

What could be wrong?
Any suggestion?

Thanks

**Comment by olebole** [comment, marked as answer]
You need to specify the architecture without `<>`, and you need to use the value from the `<arch>` column in the table in the [install document](https://github.com/iraf-community/iraf/blob/master/INSTALL.md):
```
$ make linux64
```
